tests/benchdnn/graph/graph_memory.cpp

```diff
@@ -104,7 +107,10 @@ dnn_graph_mem_t::dnn_graph_mem_t(const dnn_mem_t &mem,
     }
 }
 
-int dnn_graph_mem_t::fill_mem_with_data(const dnn_mem_t &mem) {
+int dnn_graph_mem_t::fill_mem_with_data(
+        const dnn_mem_t &mem, const dnnl::engine &graph_eng) {
+    const auto &src_eng = mem.engine();
+    const auto &dst_eng = graph_eng.get();
```

Comment:
I think the better name is just `eng` since it overrides the memory engine based on the code. THe other one could be `mem_eng`.